memorywipe.py

```diff
@@ -288,7 +288,7 @@ def ata_hdparm(self):
         click.secho("You selected ATA Secure Erase...", fg="magenta")
         click.echo("Starting ATA Secure Erase using hdparm...")
         try:
-            assert _install_tool("hdparm")
```

Comment:
If assert is present here, it is always leading to the except below and terminating. So, I removed assert. Make necessary changes to the remaining code accordingly.

Reply:
Yes you're correct! There's no boolean return statement for the assert to work.